### ISSUE TITLE
Staging Review finding: Heading structure doesn't match page content

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104800]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ask VA Integration
**Product:** Ask VA
**Feature:** Ask Va

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

There are a few places in the product where the heading structure doesn't match the content that it's supposed to describe:
- On [/ask-va/category-topic-1](https://staging.va.gov/contact-us/ask-va/category-topic-1) when unauthenticated, the sign-in alert is an H2 (equal to the progress bar) rather than an H3 (child of the progress bar)
- On [/ask-va/who-is-your-question-about](https://staging.va.gov/contact-us/ask-va/who-is-your-question-about) when unauthenticated, the sign-in alert is an H2
- On [/ask-va/review-then-submit](https://staging.va.gov/contact-us/ask-va/review-then-submit) headings inside accordions are sometimes nested correctly beneath the accordion trigger and sometimes not
- Also on [/ask-va/review-then-submit](https://staging.va.gov/contact-us/ask-va/review-then-submit) if a user dismisses the dismissable info alert the heading structure meaningfully changes such that the accordions are no longer correctly nested

## Recommended action

As needed, I'm happy to provide page-by-page feedback on heading structure. But in general the best way to approach this is to go through this exercise:

Take everything on the page (text, headings, interactive elements) and make a plain text outline --- think 9th grade English class, outlining an essay. The title should be your H1, each big section of content should be labeled with an H2, each subsection within those big sections labeled with an H3, etc

Note that accordion triggers are themselves also headings and also need to fit into this heading structure.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
